core/templates/dev/head/pages/topics_and_skills_dashboard/topics_and_skills_dashboard.html

```diff
@@ -25,38 +25,35 @@ <h2 class="dashboard-title" translate="I18N_DASHBOARD_TOPICS_AND_SKILLS_DASHBOAR
         <li ng-if="topicSummaries.length > 0"
             ng-class="{'dashboard-tabs-active': activeTab === TAB_NAME_TOPICS}">
           <a class="dashboard-tabs-text"
-             ng-click="setActiveTab(TAB_NAME_TOPICS)"
-             translate="I18N_DASHBOARD_TOPICS">
+             ng-click="setActiveTab(TAB_NAME_TOPICS)"> Topics
```

Comment:
Move "Topics" to its own line (and indent by 2 from the a tag). Ditto below.

Reply:
Done.